projects/packages/videopress/src/client/admin/components/pagination/style.module.scss

```
@@ -0,0 +1,12 @@
+.wrapper {
+	display: flex;
+	align-items: center;
+
+	.navigation span {
+		display: flex;
+	}
+
+	.button:not(.selected):disabled {
```

The following suggestion ensures 32x32 dims for the buttons

```suggestion
	.button {
		width: calc( var( --spacing-base ) * 4 );
		height: calc( var( --spacing-base ) * 4 );
		line-height: calc( var( --spacing-base ) * 4 );
	}

	.button:not(.selected):disabled {
```